### PR TITLE
KFSPTS-8335 setting employee flag on payments

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/pdp/service/impl/CuPaymentFileServiceImpl.java [hunk 1]
```diff
@@ -90,18 +90,19 @@ public void loadPayments(PaymentFileLoad paymentFile, LoadPaymentStatus status,
         status.setLoadStatus(LoadPaymentStatus.LoadStatus.SUCCESS);
     }
     
-    private void setPaymentAddressFieldsForEmployeePayee(PaymentFileLoad paymentFile, PaymentGroup paymentGroup) {
-        LOG.debug("setPaymentAddressFieldsForEmployeePayee, entering");
+    private void setPaymentFieldsForEmployeePayee(PaymentFileLoad paymentFile, PaymentGroup paymentGroup) {
```
Comment (nkimble) [inline src/main/java/edu/cornell/kfs/pdp/service/impl/CuPaymentFileServiceImpl.java]: This one method is really doing multiple things now.  The setting of the address data probably needs to be performed in its own method. Thoughts?

Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/pdp/service/impl/CuPaymentFileServiceImpl.java]: let me see if I can change things around

**Comment by jhulslander** [comment]
I reorganized this per notes, this is ready for review again

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
thanks

**Pull request merged**